pkgs/development/compilers/llvm/common/libc/default.nix

```diff
@@ -58,8 +58,8 @@ stdenv'.mkDerivation (finalAttrs: {
   outputs = [ "out" ] ++ (lib.optional isFullBuild "dev");
 
   postUnpack = lib.optionalString isFullBuild ''
-    patchShebangs $sourceRoot/../$pname/hdrgen/yaml_to_classes.py
-    chmod +x $sourceRoot/../$pname/hdrgen/yaml_to_classes.py
+    patchShebangs $sourceRoot/../$pname/utils/hdrgen/main.py
+    chmod +x $sourceRoot/../$pname/utils/hdrgen/main.py
```

Comment:
This should probably be squashed into the update commit?

Reply:
Maybe, I don't think it matters too much?

Reply:
It makes bisection complicated if there are intermediate commits that don't build.  Much better to avoid them where possible.

Reply:
This has been squashed